src/Umbraco.Web.UI.Client/src/common/resources/entity.resource.js

```diff
@@ -166,24 +166,17 @@ function entityResource($q, $http, umbRequestHelper) {
          */
         getByIds: function (ids, type) {
             
-            var query = "";
-            _.each(ids, function(item) {
-                query += "ids=" + item + "&";
-            });
-
-            // if ids array is empty we need a empty variable in the querystring otherwise the service returns a error
-            if (ids.length === 0) {
-                query += "ids=&";
-            }
-
-            query += "type=" + type;
+            var query = "type=" + type;

```

Comment:
Because we cannot do that unless I make yet another model binder that does both. This supports both without changing anything. So it's just gonna stay that way :)